sys/net/network_layer/fib/fib.c

```diff
@@ -625,6 +643,13 @@ static void fib_print_adress(universal_address_container_t *entry)
     uint8_t *ret = universal_address_get_address(entry, address, &addr_size);
 
     if (ret == address) {
+#ifdef MODULE_NG_IPV6_ADDR
```

Comment:
>> With the guard below I don't exactly get why there is a guard.
> elaborate please?

No function of the `ng_ipv6_addr` module is used, so why is there a guard?

>> Guarding it with MODULE_NG_IPV6_ADDR is the right thing
> I totally agree, but it doesn't seem to work.

Strange...